ExampleMod/Content/Tiles/ExampleOre.cs

```diff
@@ -29,6 +29,9 @@ public class ExampleOre : ModTile
 			// MineResist = 4f;
 			// MinPick = 200;
 		}
+
+		// Example of how to enable the Biome Sight buff to highlight this tile
+		public override bool IsTileBiomeSightable(int i, int j) => true;
```

Comment:
ExampleMod convention:
```suggestion
		public override bool IsTileBiomeSightable(int i, int j) {
			return true;
		}
```